components/switch/themes/spectrum.css

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+
+@container (--system: spectrum) {
+    .spectrum-Switch--emphasized {
+        --spectrum-switch-background-color-selected: var(--spectrum-accent-color-900);
+    }
+    .spectrum-Switch--disabled {
+        --spectrum-switch-handle-border-color: var(--spectrum-gray-400);
+        --spectrum-switch-background-color: var(--spectrum-gray-300);
+        --spectrum-switch-background-color-selected: var(--spectrum-gray-400);
+        --spectrum-switch-label-color: var(--spectrum-disabled-content-color);
+        /* disable hover as well */
+        --spectrum-switch-handle-border-color-hover: var(--spectrum-gray-400);
+        --spectrum-switch-handle-border-color-down: var(--spectrum-gray-400);
+    }
```

Comment:
It looks like these values do not change between Spectrum and Express. Can you pull these out of this file and put them into the `index.css` file?

Reply:
Interestingly removing this segment removes the styling for Express. That I will keep.
The disabled I moved to ndex.css`
```CSS
 @container (--system: spectrum) {
    .spectrum-Switch--emphasized {
        --spectrum-switch-background-color-selected: var(--spectrum-accent-color-900);
      }
}
```
<img width="564" alt="Screen Shot 2022-08-24 at 15 14 36" src="https://user-images.githubusercontent.com/52184321/186525482-57e4c81a-0ca3-4d30-8d06-21195575f814.png">



Disabled still works as you said - there is no diff between systems. 
<img width="1070" alt="Screen Shot 2022-08-24 at 15 18 12" src="https://user-images.githubusercontent.com/52184321/186525472-f3eddba0-c4c9-4408-ac67-3b5abfabf2b1.png">